spec/support/matchers/accessibility.rb

```diff
@@ -109,6 +109,108 @@ def descriptors(element)
   end
 end
 
+RSpec::Matchers.define :have_name do |name|
```

Comment:
Note: I realized this was a deeper rabbit hole than I imagined it would be after already pretty far down the rabbit hole 😅 

The reason for this is that Capybara's "label" checks are pretty forgiving, and don't require exactness. They also don't account for things like `aria-hidden` elements within the label. So a naive check for a label containing just "Text or voice message" would pass on `main` and not be sufficient for protecting against regressions.